Tests/Common/Data/WickoConsolidatorTests.cs

```diff
@@ -28,559 +28,559 @@ public class WickoConsolidatorTests
         [Test]
         public void OutputTypeIsRenkoBar()
         {
-            var consolidator = new WickoConsolidator(10.0m, x => x.Value);
+            var _consolidator = new WickoConsolidator(10.0m, x => x.Value);
 
```

Comment:
We do not use the underscore prefix for local variables.
Please look at our contributing guidelines here:
https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md#code-style-and-testing